src/invest_reports/utils.py

```diff
@@ -306,7 +316,35 @@ def geometamaker_load(filepath):
     return geometamaker.geometamaker.RESOURCE_MODELS[yaml_dict['type']](**yaml_dict)
 
 
-STATS_LIST = ['STATISTICS_VALID_PERCENT', 'STATISTICS_MINIMUM', 'STATISTICS_MAXIMUM', 'STATISTICS_MEAN']
+STATS_LIST = [
+    ('STATISTICS_MINIMUM', 'Minimum'),
+    ('STATISTICS_MAXIMUM', 'Maximum'),
+    ('STATISTICS_MEAN', 'Mean'),
+    ('STATISTICS_VALID_PERCENT', 'Valid percent'),
+]
+
+
+def _build_stats_table_row(resource, band):
+    # Python "general" format limits precision (default: 6 significant digits)
+    # and uses scientific notation where appropriate.
+    pandas.set_option('display.float_format', '{:G}'.format)
```

Comment:
neat! I think this a global option. If we always want to use if for all dataframes, maybe we should set it at the top-level of the module? or even the package? That way it's a little more obvious that it has a global effect. Alternatively, maybe we reset the options when we're done using them? https://pandas.pydata.org/pandas-docs/stable/user_guide/options.html#options